pkg/kp/pcstore/pcstoretest/fake_pcstore.go

```diff
@@ -56,3 +57,19 @@ func (p *FakePCStore) Delete(id fields.ID) error {
 	delete(p.podClusters, id)
 	return nil
 }
+
+func (p *FakePCStore) FindWhereLabeled(
```

Comment:
or maybe this fake store could just be the real store but with a fake consul api client put in? that'd be cool but can be a different PR